.github/workflows/update_pixi_lockfile.yml

```diff
@@ -0,0 +1,53 @@
+name: Update pixi lockfile
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 5 1 * *
+
+jobs:
+  pixi-update:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          lfs: true
```

Comment:
Question: does this already perform the lfs pull / initialization? Is the `git lfs install` necessary in the step below?

Reply:
Solved in https://github.com/ami-iit/jaxsim/pull/221/commits/667d4f3edddad5e23b8b2ccbc34eef4c5e04b52f.

See https://github.com/flferretti/jaxsim/pull/14